src/platform/packages/shared/response-ops/rules-apis/hooks/use_get_internal_rule_types_query.ts

```diff
@@ -18,5 +18,6 @@ export const useGetInternalRuleTypesQuery = ({ http }: { http: HttpStart }) => {
   return useQuery({
     queryKey: getKey(),
     queryFn: () => getInternalRuleTypes({ http }),
+    staleTime: Infinity,
```

Comment:
Available rule types don't change in a single session, caching them indefinitely (we should probably do the same with the public rule types endpoint and other APIs too)